zgw/openproduct/src/main/kotlin/nl/nlportal/openproduct/autoconfigure/OpenProductDmnConfiguration.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015-2025 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.nlportal.openproduct.autoconfigure
+
+import nl.nlportal.core.ssl.Ssl
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "nl-portal.config.openproduct.dmn", ignoreUnknownFields = true)
```

Comment:
Could we have the dmn config be a child of the OpenProductModuleConfiguration properties e.g. `nl-portal.config.openproduct.proprties.dmn`. It makes more sense to me since this dmn config and code is only used by openproduct.